Source/WebCore/platform/graphics/cg/UTIRegistry.cpp

```diff
@@ -57,10 +57,6 @@ const MemoryCompactLookupOnlyRobinHoodHashSet<String>& defaultSupportedImageType
             "public.webp"_s,
             "com.google.webp"_s,
             "org.webmproject.webp"_s,
-#endif
-#if HAVE(AVIF) || USE(AVIF)
-            "public.avif"_s,
-            "public.avis"_s,
```

Comment:
I think we should keep these entries in this list. This is the list of the default image types WebKit supports for CG ports. I think adding USE(AVIF) was wrong  and should be removed.

Reply:
Fixed.